controllers/control-plane/kcp_controller_helper_test.go

```diff
@@ -210,8 +211,12 @@ func updateKymaCRD(clnt client.Client) (*v1extensions.CustomResourceDefinition,
 		client.FieldOwner(v1beta2.OperatorName)); err != nil {
 		return nil, err
 	}
-
 	crd, err = fetchCrd(clnt, v1beta2.KymaKind)
+	kymaCrdName := fmt.Sprintf("%s.%s", v1beta2.KymaKind.Plural(), v1beta2.GroupVersion.Group)
```

Comment:
why we need this cache in test?

Reply:
because here I am updating the KCP crd for the test so I need to update it manually